code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -725,7 +725,11 @@
 	if(SM.sentience_type != animal_type)
 		to_chat(user, "<span class='warning'>You cannot transfer your consciousness to [SM].</span>" )
 		return ..()
-	if(jobban_isbanned(user, ROLE_ALIEN)) //ideally sentience and trasnference potions should be their own unique role.
+	var/jb = jobban_isbanned(user, ROLE_ALIEN)
+	if(QDELETED(src) || QDELETED(M) || QDELETED(user))
```

Comment:
![starecomputer gun](https://user-images.githubusercontent.com/4343468/41198779-08ca0882-6cc9-11e8-9391-a5775cdfbdaa.jpg)

Reply:
I operate on the assumption that @kevinz000 is actively trying to harm these lines

Reply:
make qdeleted accept multiple arguments and it checks each one

then you can just do if(QDELETED(src, M, user))

Reply:
implied semantics between `&&` and `||` would be hard to deduce from that macro